bin/inx-chronicle/src/api/error.rs

```diff
@@ -153,8 +153,9 @@ impl IntoResponse for ErrorBody {
 impl From<ApiError> for ErrorBody {
     fn from(err: ApiError) -> Self {
         if let ApiError::Internal(e) = &err {
-            log::error!("{}", e);
+            log::error!("Internal API error: {}", e);
```

Comment:
Is it a good idea to log something inside `from`?

Reply:
Yes